Libraries/VirtualFunctions/VirtualFunctions.h

```diff
@@ -0,0 +1,312 @@
+// Code made by the RoBorregos team in 2020 for the RoboCup JR. Rescue Maze category.
```

Comment:
Make a .cpp file out of this too.
Do something in the .h like:
namespace VirtualFunctions {
// declaring all the functions
}
and in the .cpp like:
namespace VirtualFunctions {
//defining the functions.
}

Reply:
I think I did it correctly, but I am not completely sure.

Reply:
Yes, thank you!